pipelines.adoc

```diff
@@ -159,23 +159,58 @@ A sample `manual-pipeline-run-template.yaml` file is provided in the link:https:
 oc apply -f <collection-name>-pipeline-run.yaml
 ```
 
-== Checking the status of the pipeline run
+== Execute pipelines manually from the command line for your custom built Collections
 
-You can check the status of the pipeline run from the OKD console,
-command line, or Tekton dashboard.
+If you have your own custom build collection image and want to run any pipelines against it instead of out of box kabanero collections, then you need to follow below steps as per the required scenarios.
 
+=== Setup of container registry url for the custom Collection image
 
-===  Ashish Execute pipelines manually from the command line for your custom built Collections
+By default, the pipelines pull the collection images for docker hub. If you are publishing your collection images to any other repository, follow the instructions below to configure the custom repository that the pipelines should pull the collection images from.
 
-If you have your own custom build collection image and want to run any pipelines against it instead of out of box kabanero collections, then you need to follow below steps as per the required scenarios.
+. After you have cloned kabanero-pipelines repository find the configmap template file `collection-image-registry-map.yaml` and add your container registry URL in it for the key `default-collection-image-registry-url` 
 
-=== Setup of container registry url for the custom Collection image
 
-. By default the pipelines assume you are pulling the collection from `kabanero` so it looks for the collection image in dockerhub repository. If you want to use
+ cd kabanero-pipelines/pipelines/common/
+ vi collection-image-registry-map.yaml
+
+. Apply the above configmap file which will set your container registry.
+
+ oc apply -f collection-image-registry-map.yaml
+
+==== For Collection image stored in container registry with internal route url on the OCP cluster
+
+For internal openshift registry you need to only setup above `collection-image-registry-map.yaml` with the internal registry url.
+
+NOTE : The service account should let the pipelines pull from the internal registry without needing to configure a secret.
```

Comment:
**change**
NOTE : The service account should let the pipelines pull from the internal registry without needing to configure a secret.

**to**
In this case, the service account that is associated with the pipelines must be configured to allow the pipelines pull from the internal registry without configuring a secret.

is there somewhere we can point to for information about service accounts? ie "For more information, see x"